l10n_it_pec_messages/view/mail_view.xml

```diff
@@ -2,6 +2,24 @@
 <openerp>
 <data>
 
+
+        <record model='ir.ui.view' id='notifications_pec_tree'>
+            <field name="name">pec.notifications.tree</field>
+            <field name="model">pec.notifications</field>
```

Comment:
@eLBati  @angedras  il model pec.notification non e' stato riportato in questa PR?
https://github.com/archetipo/l10n-italy/pull/19/files#diff-2a70234f43870d373510a9598ccc028dR29